### PR TITLE
First Draft of Riato bridge docs

### PR DESCRIPTION
**Pull request by jeffersonnnn**
This is the first draft of the rialto bridge docs. please review @kingpinXD

### CODE REVIEW

File: docs/dapp/11-rialto.md [hunk 1]
```diff
@@ -1,136 +1,35 @@
...
 
-We will be using the Rialto Bridge contract CLI to deploy and interact with the contracts. Grab and install the CLI by running:
 
-```
-git clone -b v1.0.0 --depth 1 https://github.com/FindoraNetwork/chainbridge-tools \
-&& cd chainbridge-deploy/cb-sol-cli \
-&& npm install \
-&& make install
-```
+## Stages for Relayer voting
```
Comment (kingpinXD) [inline docs/dapp/11-rialto.md:22]: Please add some text for proposal status also 
https://github.com/ChainSafe/chainbridge-solidity/blob/e2f5cda38a6f4e636094d68d8f4d0edf9aab5014/contracts/Bridge.sol#L28

File: docs/dapp/11-rialto.md [hunk 2]
```diff
@@ -1,136 +1,35 @@
...
+- Executed
 
-### Rialto Bridge Vars
-```
-SRC_GATEWAY=https://prod-forge.prod.findora.org:8545/
-DST_GATEWAY=https://data-seed-prebsc-1-s1.binance.org:8545/
 
-SRC_PK="<private key on Findora>"
-DST_PK="<private key on BSC>"
+## How does this work
```
Comment (kingpinXD) [inline docs/dapp/11-rialto.md:28]: Please use an image for the process , you can borrow it from the chainbridge docs .

File: docs/dapp/11-rialto.md [hunk 3]
```diff
@@ -1,136 +1,35 @@
...
-Transfer the wrapped tokens back to the bridge. This should result in the locked tokens being freed on the source chain and returned to your account.
-```
-cb-sol-cli --url $DST_GATEWAY --privateKey $DST_PK --gasPrice 10000000000 erc20 deposit \
-    --amount 1 \
-    --dest 0 \
-    --bridge $DST_BRIDGE \
-    --recipient 0x91388a75f30065f6F1D679541C6aDc2c3ade08A8 \
-    --resourceId $RESOURCE_ID
-```
+The executed stage sis passed and finally the bridge calls the handler contract that eventually moves the data to the Token for minting
```
Comment (kingpinXD) [inline docs/dapp/11-rialto.md:35]: This should be in the form of bullet points and not a paragraph

File: docs/dapp/11-rialto.md [hunk 4]
```diff
@@ -1,136 +1,35 @@
...
-            resourceId:
-            "0x000000000000000000000000000000c76ebe4a02bbc34786d860b355f5a5ce00",
-          },
-        ],
-      },
-    ],
-  },
-};
-```
+## Contracts and definition
```
Comment (kingpinXD) [inline docs/dapp/11-rialto.md:14]: Sample json structs for the deposit event would be helpful here .

File: docs/dapp/11-rialto.md (hunk 1 above)
Comment (jeffersonnnn) [reply docs/dapp/11-rialto.md:22]: this is done.

File: docs/dapp/11-rialto.md (hunk 2 above)
Comment (jeffersonnnn) [reply docs/dapp/11-rialto.md:39]: This is done too @kingpinXD

File: docs/dapp/11-rialto.md (hunk 4 above)
Comment (jeffersonnnn) [reply docs/dapp/11-rialto.md:22]: @kingpinXD sending you a message on slack to know who i can get this from

File: docs/dapp/11-rialto.md (hunk 1 above)
Comment (kingpinXD) [reply docs/dapp/11-rialto.md:29]: Can you please add one liners for the statues , to explain what they mean ,and when users can expect to get them

File: docs/dapp/11-rialto.md (hunk 2 above)
Comment (kingpinXD) [reply docs/dapp/11-rialto.md:37]: Please try to relate the steps to the image . You could add tags such as `1` , `2` ... in the image to signify steps

**Pull request closed without merging**